app/controllers/admin/services_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Admin::ServicesController < Admin::BaseController
+  before_action :load_service, only: %i(edit update destroy)
+
+  def index
+    @services = Service.by_lastest.page(params[:page]).per Settings.per_page
+  end
+
+  def edit; end
+
+  def update
+    if @service.update service_params
+      flash[:success] = t ".success"
+      redirect_to admin_services_path
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    if @service.destroy
+      flash[:success] = t ".success"
+    else
+      flash[:error] = t ".fail"
+    end
+    redirect_to admin_services_path
+  end
+
+  private
+  def load_service
+    @service = Service.find_by id: params[:id]
+    return if @service
+    flash[:error] = t ".not_found_service"
+    redirect_to admin_services_path
+  end
+
+  def service_params
+    params.require(:service).permit :name, :status, :price, :description
```

Comment:
Viết tòan bộ params vào model service dưới dạng  hằng số
SERVICE_PARAMS = [:name, :status, :price, :description].freeze rồi gọi ra ở đây
```suggestion
    params.require(:service).permit Service::SERVICE_PARAMS
```